omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -58,6 +58,12 @@
             # cleanup clutter
             delete "#{install_dir}/etc"
         elsif osx?
+            # Remove linux specific configs
+            conf_dir = "#{install_dir}/etc"
+            delete "#{conf_dir}/conf.d/process_agent.yaml.default"
+            delete "#{conf_dir}/conf.d/file_handle.d"
+            delete "#{conf_dir}/process-agent.conf.example"
```

Comment:
since my last review, we now have a PR that removes `process_agent.yaml.default` and `process-agent.conf.example` from all platforms, so no need to remove these 2 files anymore 😬